### PR TITLE
Authorization + authentication

### PR DESCRIPTION
**Pull request by ghost**
начало положено

### CODE REVIEW

File: festival/src/main/java/com/hubses/festival/config/security/CustomUserDetailService.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                .accountNonLocked(true)
+                .credentialsNonExpired(true)
+                .enabled(true)
+                .authorities(getRoles(user.get().getRole()))
+                .build()).orElseThrow(() -> new UsernameNotFoundException("User with name " + s +" not found"));
+    }
+
+    private List<GrantedAuthority> getRoles(Role role) {
+        List<GrantedAuthority> roles = new ArrayList<>();
+        roles.add(role);
```
Comment (zzz3bra) [inline festival/src/main/java/com/hubses/festival/config/security/CustomUserDetailService.java]: Can use return `Collections.newArrayList(role);`
UPD. Or even `Collections.singletonList(role)`

File: festival/src/test/java/com/hubses/festival/FestivalApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.hubses.festival;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class FestivalApplicationTests {
```
Comment (zzz3bra) [inline festival/src/test/java/com/hubses/festival/FestivalApplicationTests.java:10]: UserService could be tested)

**Review by zzz3bra — CHANGES_REQUESTED** [review]
- Fix SonarLint red errors
- Format code according to https://google.github.io/styleguide/javaguide.html#s4-formatting
- Fix commented code :)

**Comment by vitalics** [comment]
2 reviewers - minimum for each branch

**Comment by vitalics** [comment]
2 reviewers - minimum for each branch, fix it

**Review by zzz3bra — APPROVED** [review]

**Review by DrJekill352 — APPROVED** [review]

**Pull request merged**